proto/proto/store/rpc.proto

```diff
@@ -231,15 +242,23 @@ message AccountStorageDetails {
 // CHECK NULLIFIERS
 // ================================================================================================
 
-// List of nullifiers to return proofs for.
+// List of nullifiers to check for inclusion or non-inclusion proofs.
 message NullifierList {
-    // List of nullifiers to return proofs for.
+    // List of nullifiers to return proofs for. Each nullifier is a Digest (4 field elements).
     repeated primitives.Digest nullifiers = 1;
 }
```

Comment:
I wonder if we should allow users to query historical nullifier states - similar to what we have for accounts with `AccountTreeWithHistory`. Not a high priority, but given that we have most of the structure working already, shouldn't be too difficult to implement.

Let's create an issue for this.